modules/ROOT/pages/_partials/afm-concepts.adoc

```diff
@@ -0,0 +1,73 @@
+// Reused in index.adoc and specific feature topics
+
+// tag::afm-monitors[]
+
+A monitor is a repeating schedule of one or more tests. Monitors might also be referred to as test suites. 
+
+// end::afm-monitors[]
+
+// tag::afm-tests[]
+
+Tests are run in monitors on a defined schedule. Test cases are defined using Behavior Driven Development (BDD) language, a common, declarative, and easy-to-use behavior-description language. 
```

Comment:
```suggestion
Tests run in monitors on a defined schedule. Test cases are defined using Behavior Driven Development (BDD) language, a common, declarative, and easy-to-use behavior-description language. 
```

Reply:
Can you make this more active - "You define test cases by using Behavior Driven Development...

Reply:
If done through the UI, the BDD is generated. They can edit it manually, but the passive construct is actually more accurate here.